RecoParticleFlow/PFTracking/src/PFTrackAlgoTools.cc

```diff
@@ -36,7 +36,7 @@ namespace PFTrackAlgoTools {
     case reco::TrackBase::hltIterX:
       return  cuts[0];
     default:
-      return hltIterativeTracking ? cuts[6]:cuts[0];
+      return hltIterativeTracking ? cuts.at(5):cuts[0];
```

Comment:
@bachtis @fwyzard is it correct to set this to 5 for hlt highest iteration while we use 0 for the offline tracking highest  (or not implemented yet) iteration?

Maybe an exception or something that means no cut should be placed instead.